languages/python/templates/README.mustache

```diff
@@ -1,3 +1,4 @@
+{{! This file is picked from the v5.1.0 release of openapi-generator(https://github.com/OpenAPITools/openapi-generator/blob/v5.1.0/modules/openapi-generator/src/main/resources/Java/README.mustache) }}
```

Comment:
Looks like there are no changes in this file. Can you please remove this template?